app/tests/content/test_registration_integration.py

```diff
@@ -748,50 +748,50 @@ def test_delete_own_registration_as_member_when_no_users_on_wait_are_in_a_priori
 
 
 @pytest.mark.django_db
-def test_that_users_cannot_register_when_has_unanswered_evaluations(api_client, user):
+def test_that_users_cannot_register_when_has_unanswered_evaluations(api_client, member):
```

Comment:
Oppdater tittel til members istedenfor users. Samme for de andre testene også
```suggestion
def test_that_members_cannot_register_when_has_unanswered_evaluations(api_client, member):
```

Reply:
Fikset